ChainReactionBindings/Bindings/SearchPageBindings.cs

```diff
@@ -116,5 +114,60 @@ public void ClickBuyButtonOnItem()
 			Thread.Sleep(TimeSpan.FromSeconds(1));
 		}
 
+		[When(@"I click gender ""(.*)"" on search page")]
+		public void ClickGenderOnSearchPage(string gender)
+		{
+			page.GenderClick(gender);
+		}
+
+		[When(@"I click brand ""(.*)"" on search page")]
+		public void ClickBrandOnSearchPage(string brand)
+		{
+			page.BrandClick(brand);
+		}
+
+		[When(@"I click colour ""(.*)"" on search page")]
+		public void ClickColourOnSearchPage(string colour)
+		{
+			page.ColourClick(colour);
+		}
+
+		[Then(@"I see items include colour ""(.*)""")]
+		public void ThenISeeItemsIncludeColour(string colour)
+		{
+			var itemsDesc = page.ItemsCountElement.FindElements(By.ClassName(colour.ToLower()));
+		}
+
+		[When(@"I enter price from ""(.*)"" to ""(.*)"" on search page and click Go button")]
+		public void EnterPriceFromToOnSearchRageAndClickGoButton(string from, string to)
+		{
+			page.FromElement.SendKeys(from);

```

Comment:
Can we hide these three lines in some nice function inside the page object?

Reply:
done